fplll/defs.h

```diff
@@ -179,6 +181,15 @@ enum LLLMethod
 
 const char *const LLL_METHOD_STR[6] = {"wrapper", "proved", "heuristic", "fast"};
 
+enum HNFMethod
+{
+  HM_CLASSIC     = 0,
+  HM_XGCD        = 1,
+  HM_PERNETSTEIN = 2
+};
+
+const char *const HNF_METHOD_STR[6] = {"classic", "xgcd", "pernet-stein"};
```

Comment:
These aren't actually all implemented, right? We shouldn't advertise something which is not implemented yet. Also the array is too big.

Reply:
It's true, I used 6 like the size of LLL_METHOD_STR which has size 6 too while it only had 4 enums defined above. I am not sure why to be honest I should have asked before, but I'll fix that to 3 while it might grow up later.